tests/Unit/Modifiers/AggregateFacilityUseProviderModifierTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace OpenActive\Models\Tests\Unit\Modifiers;
+
+use OpenActive\Modifiers\AggregateFacilityUseProviderModifier;
+use PHPUnit\Framework\TestCase;
+
+class AggregateFacilityUseProviderModifierTest extends TestCase
+{
+    public function testModifierRemovesProvider()
```

Comment:
This test is great! If we could also have a test to make sure that it **doesn't** remove the provider in the opposite circumstances, that would be great

Reply:
Resolved by: https://github.com/openactive/models-php/pull/109/commits/87f4b65d85adabf8af090926c3317858341dd609